src/gas_price_increase.rs

```diff
@@ -10,8 +10,8 @@ const MIN_GAS_PRICE_INCREASE_FACTOR: f64 = 1.125 * (1.0 + f64::EPSILON);
 /// The minimum gas price that allows a new transaction to replace an older one.
 pub fn minimum_increase(previous_gas_price: EstimatedGasPrice) -> EstimatedGasPrice {
     previous_gas_price
-        .bump_cap(MIN_GAS_PRICE_INCREASE_FACTOR)
-        .ceil_cap()
+        .bump(MIN_GAS_PRICE_INCREASE_FACTOR)
+        .ceil()
```

Comment:
I see that we went from `bump_cap` to just `bump`... does this mean that our bumps are no longer capped? Or does the old functions have the same implementation just different names... 

Just want to make sure we aren't bumping without a cap...

Reply:
Previously, `bump_cap` and `ceil_cap` were mutating only `EstimatedGasPrice::max_fee_per_gas`. In the meantime, I figured out that it does make more sense to mutate also `EstimatedGasPrice::max_priority_fee_per_gas`, because this way it actually increases `EstimatedGasPrice` as a whole. In other words, there are cases when, by increasing only `EstimatedGasPrice::max_fee_per_gas` we don't increase our chances of our transaction being mined, which is a primary purpose of the `bump`.
That said, names are changed according to the implementation change.